dbt_subprojects/tokens/models/transfers_and_balances/bitcoin/tokens_bitcoin_net_transfers_daily_asset.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+        schema = 'tokens_bitcoin'
+        , alias = 'net_transfers_daily_asset'
+        , materialized = 'incremental'
```

Comment:
```suggestion
        , materialized = 'view'
```

Reply:
lets try this as a view, as the underlying table is materialized already and we aren't doing any complex joins to slow down performance on the view. this view should run at same speed as querying table directly